recipes/dd4hep/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "dd4hep" %}
+{% set version = "01-13" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version.split('-')|map('int')|join('.') }}"
+
+source:
+  url: https://github.com/AIDASoft/DD4hep/archive/v{{ version }}.zip
+  sha256: 945a04cd04e9af23cdd4ba29a41bbff0fc388a66e5b640eb3f633596f130573d
+  folder: source
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('dd4hep', max_pin='x.x') }}
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    - cmake
+    - make
+
+  host:
+    - xorg-libxfixes  # [linux]
+    - root_base
+    - root-binaries
+    - root-dependencies
+    - geant4
+    - xerces-c
+    - boost-cpp
+    - python
+#   - assimp
+
+  run:
+    - Boost
```

This should also be `boost-cpp` if linking to `boost`. If dd4hep only uses head-only libraries from Boost it can be removed entirely.
```suggestion
    - boost-cpp
```